codecov.yml

```diff
@@ -0,0 +1,2 @@
+codecov:
+  token: 027abd19-69ad-4fa9-9f0a-30ec054490f6
```

Comment:
Should we leave this token out in the open like this?
Can we move it to an environment variable?

Reply:
> Should we leave this token out in the open like this?
It was just copied from `xblock-html`, which is public, so it's not a secret.

We're not using this, so it's safe to be removed.